templates/libraries/okhttp-gson/pojo.mustache

```diff
@@ -553,6 +553,20 @@ public class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{{#vendorExtens
       }
       {{/required}}
       {{/isModel}}
+      {{#isEnum}}
+      {{#required}}
```

Comment:
Should we have similar concerns about required readOnly and writeOnly schema properties?

https://github.com/equinix-labs/metal-go/pull/132#discussion_r1337801820

Reply:
This PR is making a small update to the `validateJsonElement` section of the template, and the added code aligns with the validation that already exists for other properties.

Reply:
For reference/comparison, both this block and the subsequent `isEnumRef` block are essentially copy-pastes of the preceding `isModel` block: https://github.com/equinix-labs/metal-java/pull/163/commits/ddc37db375e481dfd1b2c317d30886dbc684a419#diff-f6e410ad0dea301581f6b3054b9f8fb2bcdb8f05807630eefbdddca891cd1371R544-R555